src/frontend/src/stylesheets/layouts/sidebar/sidebar.module.css

```diff
@@ -14,10 +14,11 @@
 }
 
 .versionNumber {
-  position: absolute;
+  position: relative;
   text-align: center;
   color: #ffffff;
-  width: 100%;
   bottom: 0;
+  width: 100%;
   height: 6em;
```

Comment:
delete lines 20, 22, and 23, and make sure to add in a marginBottom (like 20-30ish) so the version number isn't right on the bottom of the screen